### PR TITLE
pfstools: 1.8.5 -> 2.0.5

### PR DESCRIPTION
**Pull request by juliendehos**
###### Things done
- [ ] Tested using sandboxing
  ([nix.useChroot](http://nixos.org/nixos/manual/options.html#opt-nix.useChroot) on NixOS,
    or option `build-use-chroot` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [X] NixOS
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/graphics/pfstools/default.nix [hunk 1]
```diff
@@ -1,24 +1,28 @@
...
-  buildInputs = [ libtiff openexr imagemagick libjpeg qt4 mesa freeglut
-    bzip2 libX11 libpng expat ];
+  buildInputs = [ openexr zlib imagemagick mesa freeglut fftwFloat fftw gsl libexif perl opencv qt4 ];
 
-  nativeBuildInputs = [ pkgconfig ];
+  nativeBuildInputs = [ cmake pkgconfig ];
 
-  meta = {
-    homepage = http://pfstools.sourceforge.net/;
+  patches = [ ./threads.patch ./pfstools.patch ];
```
Comment (garbas) [inline pkgs/tools/graphics/pfstools/default.nix]: @juliendehos would it be possible to fetch this patches (using fetchurl) from some url? this way next person will know where to look for newer patches.

File: pkgs/tools/graphics/pfstools/default.nix [hunk 2]
```diff
@@ -1,24 +1,28 @@
...
+
+  meta = with stdenv.lib; {
+    homepage = "http://pfstools.sourceforge.net/";
     description = "Toolkit for manipulation of HDR images";
-    license = "GPL";
+    platforms = platforms.linux;
+    license = licenses.lgpl2;
```
Comment (garbas) [inline pkgs/tools/graphics/pfstools/default.nix]: @juliendehos could you add yourself as a maintainer?

File: pkgs/tools/graphics/pfstools/default.nix (hunk 1 above)
Comment (juliendehos) [inline pkgs/tools/graphics/pfstools/default.nix]: No, I wrote these patches myself. They are very simple so updating should not be a problem.

File: pkgs/tools/graphics/pfstools/default.nix (hunk 2 above)
Comment (juliendehos) [inline pkgs/tools/graphics/pfstools/default.nix]: Ok. Done.

File: pkgs/tools/graphics/pfstools/default.nix [hunk 3]
```diff
@@ -1,24 +1,29 @@
...
 
 stdenv.mkDerivation rec {
-  name = "pfstools-1.8.5";
+  name = "pfstools";
+  version = "2.0.5";
 
   src = fetchurl {
-    url = "mirror://sourceforge/pfstools/${name}.tar.gz";
-    sha256 = "01kk2r8afrb3vrhm8abfjdhhan97lzpapc4n8w1mpdp3kv9miy9c";
+    url = "http://downloads.sourceforge.net/project/pfstools/pfstools/${version}/${name}-${version}.tgz";
```
Comment (joachifm) [inline pkgs/tools/graphics/pfstools/default.nix]: Please change back to using a mirror url; downloads.sourceforge.net is now the first mirror tried, if that's a concern.

**Comment by LnL7** [comment]
This includes binaries and development headers, is it worth putting those in separate outputs?

File: pkgs/tools/graphics/pfstools/default.nix (hunk 3 above)
Comment (juliendehos) [inline pkgs/tools/graphics/pfstools/default.nix]: Done.

**Comment by garbas** [comment]
merged manually [here](https://github.com/NixOS/nixpkgs/commit/d49902e03a0c2adefa36ef8865bc9540c74697f9)  ... i;ve also added "dev" output as suggested by @LnL7

**Comment by juliendehos** [comment]
ok, thanks

**Pull request closed without merging**